src/Layout/Presentation/SingleDistrict/SingleDistrict.tsx

```diff
@@ -44,6 +49,11 @@ export class SingleDistrict extends React.Component<SingleDistrictProps, {}> {
         return (
             <React.Fragment>
                 <h1 className="h1">{this.props.districtSelected}</h1>
+                <p>
+                    Sistemandat i {this.props.districtSelected} gikk til {vulnerable.winner.partyCode}.{" "}
```

Comment:
Why is there a {" "} at the end of this line? You can use <br/> to break the line.

Reply:
I cleaned this up a bit, it should be better now.